draft-ietf-quic-transport.md

```diff
@@ -1422,6 +1422,16 @@ server's new preferred_address value in the handshake.
 A server MUST either reject 0-RTT data or abort a handshake if the implied
 values for transport parameters cannot be supported.
 
+A client MUST NOT use updated values it learns either from the server's updated
+transport parameters or frames carried in 1-RTT when sending frames in 0-RTT
+packets.  Updated values of transport parameters from the handshake only apply
+to 1-RTT packets.  For instance, flow control limits from remembered transport
+parameters apply to all 0-RTT packets even if those values are increased by the
+handshake; increased limits from frames cannot be applied, because the client
```

Comment:
```suggestion
handshake.  Increased flow control limits from frames cannot be applied, because the client
```

Reply:
I might even suggest dropping the second half, after "increased limits from frames".